internal/batches/workspace/volume_workspace.go

```diff
@@ -219,7 +229,8 @@ var _ Workspace = &dockerVolumeWorkspace{}
 
 func (w *dockerVolumeWorkspace) Close(ctx context.Context) error {
 	// Cleanup here is easy: we just get rid of the Docker volume.
-	return exec.CommandContext(ctx, "docker", "volume", "rm", w.volume).Run()
+	_ = deleteVolume(ctx, w.volume)
+	return nil
```

Comment:
```suggestion
	return deleteVolume(ctx, w.volume)
```

To keep the error reporting

Reply:
D'oh, not sure how I misread the code. I'll fix.

Reply:
Fixed, thanks for pointing out!